src/scss/design-system-interface.scss

```diff
@@ -251,11 +230,10 @@ $weight-xbold: 900;
     font-family: $primary-font;
     font-weight: $weight-bold;
     line-height: 1;
-
+    padding-bottom: 1rem;
     color: $c-dark-brown;
-
     margin: 0;
-
+    
```

Comment:
a little bit of space snuck in here 😄

Reply:
this is alright. just made a space for the media query to go after